docs/tutorials/alibabacloud.md

```diff
@@ -0,0 +1,365 @@
+# Setting up ExternalDNS for Services on Alibaba Cloud
+
+This tutorial describes how to setup ExternalDNS for usage within a Kubernetes cluster on Alibaba Cloud. Make sure to use **>=0.4** version of ExternalDNS for this tutorial
```

Comment:
I would recommend to change 0.4 to version 0.5.6. This would be the next version incl. Alibaba Cloud